aiapy/calibrate/util.py

```diff
@@ -12,40 +11,119 @@
 
 import astropy.io.ascii
 import astropy.units as u
+from astropy.io import ascii as astropy_ascii
 from astropy.table import QTable
 from astropy.time import Time
 
-import drms
 from sunpy import log
-from sunpy.net import attrs, jsoc
 
 from aiapy import _SSW_MIRRORS
 from aiapy.data._manager import manager
 from aiapy.util.decorators import validate_channel
+from aiapy.util.net import get_data_from_jsoc
 
-__all__ = ["get_correction_table", "get_error_table", "get_pointing_table"]
+__all__ = [
+    "get_correction_table",
+    "get_error_table",
+    "get_pointing_table",
+]
 
-# Default version of the degradation calibration curve to use.
-# This needs to be incremented as the calibration is updated in JSOC.
-CALIBRATION_VERSION = 10
 # Error table filename available from SSW
-AIA_ERROR_FILE = "sdo/aia/response/aia_V{}_error_table.txt"
-# Most recent version number for error tables; increment as new versions become available
-ERROR_VERSION = 3
+_AIA_ERROR_FILE = "sdo/aia/response/aia_V{}_error_table.txt"
 # URLs and SHA-256 hashes for each version of the error tables
-URL_HASH = {
+_URL_HASH_ERROR_TABLE = {
     2: (
-        [urljoin(mirror, AIA_ERROR_FILE.format(2)) for mirror in _SSW_MIRRORS],
+        [urljoin(mirror, _AIA_ERROR_FILE.format(2)) for mirror in _SSW_MIRRORS],
         "ac97ccc48057809723c27e3ef290c7d78ee35791d9054b2188baecfb5c290d0a",
     ),
     3: (
-        [urljoin(mirror, AIA_ERROR_FILE.format(3)) for mirror in _SSW_MIRRORS],
+        [urljoin(mirror, _AIA_ERROR_FILE.format(3)) for mirror in _SSW_MIRRORS],
         "66ff034923bb0fd1ad20e8f30c7d909e1a80745063957dd6010f81331acaf894",
     ),
 }
+_URL_HASH_POINTING_TABLE = (
+    "https://aia.lmsal.com/public/master_aia_pointing3h.csv",
+    "a2c80fa0ea3453c62c91f51df045ae04b771d5cbb51c6495ed56de0da2a5482e",
+)
+_URL_HASH_RESPONSE_TABLE = {
```

Comment:
Same here. I think we should go with either "response" or "correction". I realize that this is a bit weird since the files themselves in SSW are called `*_response_table.txt`, but I find this file naming convention really confusing. I believe that I explicitly referred to them as "correction tables" when I wrote the original version of all this to make it clear what they were and to avoid any confusion with the files used to compute the wavelength response functions.

Reply:
Renamed to correction tables.